mini-media-player-bundle.js

```diff
@@ -0,0 +1,1785 @@
+const directives = new WeakMap();
```

Comment:
I don't want to include the built bundle in the source. I think we should add it as an asset with every release instead.